pkg/cmd/grafana-server/server.go

```diff
@@ -163,9 +162,18 @@ func (g *GrafanaServerImpl) Run() error {
 		})
 	}
 
-	sendSystemdNotification("READY=1")
+	defer func() {
```

Comment:
@marefr I think this is a good technique for ensuring that `g.childRoutines` is waited on regardless of later error handling, but interested in your feedback :)

Reply:
I agree, but in this case I don't think it works as expected. Thinking about it some more I'm not sure if the whole grafana instance should exit if sendSystemdNotification fails, maybe easier to log instead? Looking at implementation of sendSystemdNotification seems like there's many reason that can error out, if the environment variable is not set for example.

Reply:
@marefr OK, I will look more closely.

Reply:
@marefr I think you're right about the systemd notification, I changed the logic to just warn if it fails.